serverless/components
# Remove --all flag and detect templates automatically

## What has been implemented?

Remove --all flag and detect templates automatically. PR also fixes a minor bug when running remove (the "Success" message was logged twice)

With that in place, the Components CLI runs in the following cases:

- running `sls init`
- running `sls publish`
- running `sls registry`
- running inside a directory containing `serverless.yml` file of an instance
- running inside a directory containing child directories of instances and NO serverless.yml in the root
- NOT running `help` or `version`

## Steps to verify

- cd into a template directory
- instead of running `sls deploy --all` and `sls remove --all` as usual, just run `sls deploy` and `sls remove`
- the framework should detect that there are components in child directories and run them as a template.
